components/Layout.jsx

```diff
@@ -8,6 +8,17 @@ function Layout({ children }) {
   const [darkMode, setDarkMode] = useState(false);
   const [showBanner, setShowBanner] = useState(true);
 
+  useEffect(() => {
+    const storedTheme = localStorage.getItem('DARK_MODE');
+    if (storedTheme === 'true') {
+      document.body.classList.toggle('dark');
+      document.body.classList.toggle('bg-gray-800');
```

Comment:
Toggle this class on the `html` tag & give the `body` tag the class of `dark:bg-gray-800 bg-white`